src/lib.rs

```diff
@@ -24,6 +24,8 @@ pub async fn sync_ldap_users_to_zitadel(config: Config) -> Result<()> {
 	let zitadel = Zitadel::new(&config).await?;
 	let (mut ldap_client, ldap_receiver) = Ldap::new(config.clone().ldap.into(), cache);
 
+	let deactivate_only = config.feature_flags.contains(&FeatureFlag::DeactivateOnly);
```

Comment:
Small nit: We could have a `config.disable_only()` so we don't need to do this `.contains` shenanigans all the time.

Reply:
We do have such function for some other feature flag. I'd say we should add some generic method to config:
```rust
pub fn is_feature_enabled(feature_flag: FeatureFlag) -> bool {...}
```
However the ".contains shenanigans" are quite readable, so whatever, but let's make it consistent with other "is feature X enabled" expressions, explicit or hidden behind some function